docs/modules/help/pages/tickets_detail.adoc

```diff
@@ -0,0 +1,29 @@
+= Viewing a ticket
+
+[[overview]]
+== Overview
+
+When clicking on a ticket in a ticket list, the ticket detail screen appears. The screen shows the following major elements:
+
+. Ticket summary at the top, including buttons for editing and closing the ticket.
+. Ticket detail information at the right hand side.
+. Associated secrets.
+. Attached files.
+. Finally, the comments exchanged between VSHNeers and the customer.
+
+The screenshot below shows the major elements of the upper side of the screen.
+
+.Ticket Detail screen
+image::ticket-detail.png[]
+
+<1> The btn:[Edit] button gives you the possibility to change some of the information associated to the ticket.
+<2> The btn:[Close] button, when enabled, allows you to close and discard this ticket.
```

Comment:
A ticket is not "discarded" when marked as closed. I'd suggest "... allows you to mark a ticket as resolved".